Source/SwiftLintFramework/Models/RuleDescription.swift

```diff
@@ -13,18 +13,32 @@ public struct RuleDescription: Equatable {
     public let nonTriggeringExamples: [String]
     public let triggeringExamples: [String]
     public let corrections: [String: String]
+    public let aliases: Set<String>
+    public let deprecatedAliases: Set<String>
 
     public var consoleDescription: String { return "\(name) (\(identifier)): \(description)" }
 
+    public var allAliases: Set<String> {
+        return aliases.union(deprecatedAliases)
+    }
+
+    public var allIdentifiers: [String] {
+        return Array(allAliases) + [identifier]
+    }
+
     public init(identifier: String, name: String, description: String,
                 nonTriggeringExamples: [String] = [], triggeringExamples: [String] = [],
-                corrections: [String: String] = [:]) {
+                corrections: [String: String] = [:],
+                aliases: Set<String> = [],
+                deprecatedAliases: Set<String> = []) {
         self.identifier = identifier
         self.name = name
         self.description = description
         self.nonTriggeringExamples = nonTriggeringExamples
         self.triggeringExamples = triggeringExamples
         self.corrections = corrections
+        self.aliases = aliases
```

Comment:
Why are there aliases, and deprecated aliases? Are you being proactive in case we want to add non-deprecated aliases in the future? I'm not sure that's a feature we'd actually want. Unless there's another reason I'm not aware of?